scripts/run_mypy_checks.py

```diff
@@ -372,16 +371,17 @@ def install_mypy_prerequisites(install_globally):
             cmd + uextention_text, stdout=subprocess.PIPE,
             stderr=subprocess.PIPE)
         new_process.communicate()
-        _PATHS_TO_INSERT.append(os.path.join(site.USER_BASE, 'bin'))
-        mypy_exec_path = os.path.join(site.USER_BASE, 'bin', 'mypy')
+        if site.USER_BASE:
```

Comment:
Why is this if needed?

Reply:
If we check `site.USER_BASE`, we will get to know that it can't be `None` here. So, to tell the Mypy, I added an `assert` ( this `if` statement is removed ).